pxc-57/dockerdir/entrypoint.sh

```diff
@@ -92,6 +92,13 @@ function join {
 	echo "${joined%?}"
 }
 
+# if vault secret file exists we assume we need to turn on encryption
+vault_secret="/etc/mysql/vault-keyring-secret/keyring_vault.conf"
+if [ -f "$vault_secret" ]; then
+	sed -i "/\[mysqld\]/a early-plugin-load=keyring_vault.so" $CFG
```

Comment:
```suggestion
	sed -i "/\[mysqld\]/a early-plugin-load=keyring_vault.so" $CFG
	sed -i "/\[mysqld\]/a default_table_encryption=1" $CFG
```

Reply:
`default_table_encryption` does not work in PXC5.7